src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -1052,6 +1052,11 @@ typedef struct __ClientInfo {
 
     // whether to log metric or not
     BOOL logMetric;
+
+    // ------------------------------- V0 compat ----------------------
+
+    // Time that allowed to be elapsed between the metric loggings if enabled
+    UINT64 metricLoggingPeriod;

```

Comment:
If we are going to do c producer level sampling and aggregation, why does PIC need to hold metricLoggingPeriod?

Reply:
This really indicates the amount of time that should elapse for the DLOGD printout. Useful for debug logs. Previously, we were logging debug log on every key frame which is terrible if we are handling audio stream only with all key frames and hundreds of FPS.